samlsp/middleware.go

```diff
@@ -95,8 +95,8 @@ func (m *Middleware) GetAssertion(r *http.Request) (*saml.Assertion, error) {
 	assertion, err := m.ServiceProvider.ParseResponse(r, m.getPossibleRequestIDs(r))
 	if err != nil {
 		if parseErr, ok := err.(*saml.InvalidResponseError); ok {
-			m.ServiceProvider.Logger.Printf("RESPONSE: ===\n%s\n===\nNOW: %s\nERROR: %s",
-				parseErr.Response, parseErr.Now, parseErr.PrivateErr)
+			m.ServiceProvider.Logger.Printf("Error parsing saml assertion: %s with error: %s",
```

Comment:
Maybe we should log parseErr.Now, since it is the time the response was actually evaluated.  It could save us some uncertainty versus relying on the log message timestamp.